modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -9,7 +9,7 @@
 Instead of logging in to the {product-title} registry from within the cluster,
 you can gain external access to it by exposing it with a route. This allows you
 to log in to the registry from outside the cluster using the route address, and
-to tag and push images using the route host.
+to tag and push images to an existing project using the route host.
```

Comment:
"By using" the route host or using route host information as part of the tag + push process?

ISG:

> ⚠️ using _noun_
> To avoid ambiguity, replace this gerund with either "by using" or "that uses".
> 
> To refer to something "using" one thing or another is often ambiguous. For example, the sentence "Back up the server using ABC" might mean "Back up the server by using ABC" or "Back up the server that uses ABC". This advice also applies to gerunds other than "using". For more information, see the style and grammar tips in the Global audiences topic in IBM Style and the section about "Clarity" in "Developing Quality Technical Information".

Reply:
I have made this change as this does sound a bit ambiguous.

Reply:
> "By using" the route host or using route host information as part of the tag + push process?
> 
> ISG:
> 
> > warning using _noun_
> > To avoid ambiguity, replace this gerund with either "by using" or "that uses".
> > To refer to something "using" one thing or another is often ambiguous. For example, the sentence "Back up the server using ABC" might mean "Back up the server by using ABC" or "Back up the server that uses ABC". This advice also applies to gerunds other than "using". For more information, see the style and grammar tips in the Global audiences topic in IBM Style and the section about "Clarity" in "Developing Quality Technical Information".

I have made this change as this does sound a bit ambiguous.